### PR TITLE
CPDP-1449: remove supporting evidence technical debt

### PR DESCRIPTION
**Pull request by mikail-khan**
- This PR also includes the FE changes made for CPDP-1593

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/supportingevidence/SupportingEvidenceController.scala [hunk 1]
```diff
@@ -93,30 +83,22 @@ class SupportingEvidenceController @Inject() (
...
             maybeSupportingDocumentsAnswers.fold[Future[Result]](
-              f(i, c.cgtReference, s, r, IncompleteSupportingEvidenceAnswers.empty)
-            )(f(i, c.cgtReference, s, r, _))
-
-          case i: DraftSingleIndirectDisposalReturn =>
-            i.supportingEvidenceAnswers.fold[Future[Result]](
-              f(i.id, c.cgtReference, s, r, IncompleteSupportingEvidenceAnswers.empty)
-            )(f(i.id, c.cgtReference, s, r, _))
+              f(s, r, IncompleteSupportingEvidenceAnswers.empty)
+            )(f(s, r, _))
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/supportingevidence/SupportingEvidenceController.scala:101]: we need a case for `DraftSingleIndirectDisposalReturn` as well to be able to upload files on a single indirect disposal journey

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/upscan/UpscanResult.scala [hunk 2]
```diff
@@ -15,7 +15,7 @@
  */
 
 package uk.gov.hmrc.cgtpropertydisposalsfrontend.models.upscan
-
+//TODO: Take out - not used
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/upscan/UpscanResult.scala:18]: is that something you were planning on doing in this PR or a later PR?

Comment (mikail-khan) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/upscan/UpscanResult.scala:18]: have taken that out

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/supportingevidence/SupportingEvidenceController.scala (hunk 1 above)
Comment (mikail-khan) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/supportingevidence/SupportingEvidenceController.scala:101]: Oh, is that one of the new types you mentioned last week that have been introduced?

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/supportingevidence/SupportingEvidenceController.scala:101]: yeah, to accommodate for the new shape of the return for indirect disposals. There'll probably be another one for mixed use later on

Comment (mikail-khan) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/supportingevidence/SupportingEvidenceController.scala:101]: fixed

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**